docs/guides/update_versions.md

```diff
@@ -1,26 +1,25 @@
 ---
-title: Updating versions of Rocky
+title: Performing unsupported upgrades between Major Rocky Linux releases
 author: Steven Spencer
 contributors:
 ---
 
-(OR **How to duplicate a machine install**)
 
 ## Introduction
 
 Since day one of the Rocky Linux project, there are those who have asked: ==How do you upgrade from CentOS 7 to Rocky 8, or Rocky 8 to Rocky 9?== The answer is always the same: **The project does not support in-place upgrades of one major version to another major version. You need to reinstall to move to the next major version.** To be clear, this **IS** the correct answer. This document hopes to give users a method of moving from one major version to the next, using the correct Rocky supported procedure of a fresh installation.
 
 !!! note "Caveats"
 
-    Even given this procedure, many things can go wrong when going from an older version of one operating system (OS) to a newer version of the same or different OS. Programs become obsolete, replaced by the maintainers as completely different package names, or the names simply do not match up from one OS to the next. In addition, you need to know about your machine's repositories and verify that they exist for the new OS. If going from a much older version to a much newer version, ensure that your CPU and other machine requirements match the new version. For this reason, you must remain cautious and note any errors or problems while performing this procedure. Here, the author has used Rocky Linux 8 as the old version and Rocky Linux 9 as the new major version. The formulation of all examples uses these two versions. Always proceed at your own risk.
+    Even given this procedure, many things can go wrong when going from an older version of one operating system (OS) to a newer version of the same or different OS. Programs become obsolete, replaced by the maintainers with completely different package names, or the names simply do not match up from one OS to the next. In addition, you need to know about your machine's software repositories and also verify that they are still functional for the new OS. If going from a much older version to a much newer version, ensure that your CPU and other machine requirements match the new version. For these and many other reasons, you must remain cautious and note any errors or problems while performing this procedure. Here, the author has used Rocky Linux 8 as the old version and Rocky Linux 9 as the new major version. The formulation of all examples uses these two versions. Always proceed at your own risk.
 
 ## Example old machine
 
-The old machine used here is Rocky Linux 8. In the example machine, the installation of the Extra Packages for Enterprise Linux (EPEL) repository and a few packages exist.
+The old machine used here is Rocky Linux 8. In the example machine, the package providing the Extra Packages for Enterprise Linux (EPEL) repository is installed, along with a a few other packages.
```

Comment:
"The old machine used here is Rocky Linux 8. In the example machine, the package providing the Extra Packages for Enterprise Linux (EPEL) repository is installed, along with a a few other packages." introduces passive voice with "is installed"